Sources/UtiliKit/ActiveLabel/ActiveLabel.swift

```diff
@@ -70,34 +79,69 @@ class ActiveLabel: UILabel {
+        didSet {
+            configurationChanged()
+        }
+    }
     /// Trailing Inset for the last activity line. Default is 0.
-    @IBInspectable public var finalLineTrailingInset: CGFloat = 0
+    @IBInspectable public var finalLineTrailingInset: CGFloat = Configuration.default.finalLineTrailingInset {
+        didSet {
+            configurationChanged()
+        }
+    }
     /// Line Length in points for the last activity line. If `finalLineTralingInset` is set to greater than 0 this value is not used. Default is 0.
-    @IBInspectable public var finalLineLength: CGFloat = 0
+    @IBInspectable public var finalLineLength: CGFloat = Configuration.default.finalLineLength {
+        didSet {
+            configurationChanged()
+        }
+    }
     /// This color is the darkest area of the line seen during activity animation. Default is (233,231,237) Gray.
-    @IBInspectable public var loadingViewColor: UIColor = ActiveLabel.loadingGray
+    @IBInspectable public var loadingViewColor: UIColor = Configuration.default.loadingViewColor {
+        didSet {
+            configurationChanged()
+        }
+    }
     /// The height of each activity line. Default is 8.
-    @IBInspectable public var loadingLineHeight: CGFloat = 8
+    @IBInspectable public var loadingLineHeight: CGFloat = Configuration.default.loadingLineHeight {
+        didSet {
+            configurationChanged()
+        }
+    }
     /// Vertical spacing between each activity line when 2 or more lines are displayed. Default is 14.
-    @IBInspectable public var loadingLineVerticalSpacing: CGFloat = 14
+    @IBInspectable public var loadingLineVerticalSpacing: CGFloat = Configuration.default.loadingLineVerticalSpacing {
+        didSet {
+            configurationChanged()
+        }
+    }
     /// The duration of the gradient animation applied to the activity lines. Default is 2.4.
-    @IBInspectable public var loadingAnimationDuration: Double = 2.4
+    @IBInspectable public var loadingAnimationDuration: Double = Configuration.default.loadingAnimationDuration {
+        didSet {
+            configurationChanged()
+        }
+    }
     /// The delay that is applied before each animation begins. Default is 0.4.
-    @IBInspectable public var loadingAnimationDelay: Double = 0.4
-    /// Shows the loading views in the Storyboard by default since text of a label can't be nil in a Storybaord. Default is true.
-    @IBInspectable private var showLoadingViewsInStoryboard: Bool = true
+    @IBInspectable public var loadingAnimationDelay: Double = Configuration.default.loadingAnimationDelay {
+        didSet {
+            configurationChanged()
+        }
+    }
+//    /// Shows the loading views in the Storyboard by default since text of a label can't be nil in a Storybaord. Default is true.
+//    @IBInspectable private var showLoadingViewsInStoryboard: Bool = true
+//
```

Comment:
Looks like there's some properties and comments that are commented out.

Reply:
Oops. I'll get those removed.